database/sql/V020__extend_orders_with_is_liquidity_order.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE orders ADD is_liquidity_order boolean NOT NULL DEFAULT false;
+-- So far every liquidity order was also a partially fillable order and all
+-- partially fillable orders were liquidity orders.
+-- We use this fact to initialize is_liquidity_order for existing orders correctly.
+UPDATE orders SET is_liquidity_order = partially_fillable WHERE is_liquidity_order = false;
```

Comment:
Looked through the Postgres docs a bit. I think we want
```
ALTER TABLE orders  ALTER COLUMN is_liquidity_order DROP DEFAULT;
```
at the end of the migration. Otherwise we will have set the false default set forevery which will make Postgres use it for all future insert operations that don't specify the column. We don't want this behavior we just need the default temporarily before the new value is calculated in the second command.

The second command could also be simplified to this I believe
```
UPDATE orders SET is_liquidity_order = partially_fillable
```